modules/landing_zone/main.tf

```
@@ -1,50 +1,50 @@
 resource "null_resource" "landing_zone_config" {
   triggers = {
-    command = "${var.landing_zone_command}"
+    command    = "${var.landing_zone_command}"
+    providers  = "${md5(jsonencode(var.landing_zone_providers))}"
     components = "${md5(jsonencode(var.landing_zone_components))}"
-    region = "${var.region}"
-    account_id = "${var.account_id}"
   }
 
   "provisioner" "local-exec" {
     when    = "create"
     command = "sh ${path.module}/scripts/config.sh"
+
     environment = {
-      ROOT_PATH = "${var.root_path}"
-      COMMAND = "${var.landing_zone_command}"
+      ROOT_PATH  = "${var.root_path}"
+      COMMAND    = "${var.landing_zone_command}"
+      PROVIDERS  = "${jsonencode(var.landing_zone_providers)}"
       COMPONENTS = "${jsonencode(var.landing_zone_components)}"
-      REGION = "${var.region}"
-      ACCOUNT_ID = "${var.account_id}"
     }
   }
 
   provisioner "local-exec" {
     when    = "destroy"
     command = "python ${path.module}/scripts/remove_config.py"
+
     environment = {
-      root = "${var.root_path}"
-      components = "${jsonencode(var.landing_zone_components)}"
+      ROOT_PATH  = "${var.root_path}"
+      COMPONENTS = "${jsonencode(var.landing_zone_components)}"
     }
   }
 }
 
 resource "null_resource" "landing_zone_apply" {
   depends_on = ["null_resource.landing_zone_config"]
+
   triggers = {
-    command = "${var.landing_zone_command}"
+    command    = "${var.landing_zone_command}"
     components = "${md5(jsonencode(var.landing_zone_components))}"
-    timestamp = "${timestamp()}"
+    timestamp  = "${timestamp()}"
   }
 
   "provisioner" "local-exec" {
```

Argument names should not be quoted
```suggestion
   provisioner "local-exec" {
```